engine/src/main/java/org/terasology/rendering/nui/layouts/CardLayout.java

```diff
@@ -56,15 +84,31 @@ public void removeWidget(UIWidget widget) {
         }
     }
 
+    /**
+     * Sets the currently displayed widget
+     *
+     * @param id The id of the {@code UIWidget} to be displayed
+     */
     public void setDisplayedCard(String id) {
         defaultCard = id;
     }
 
+    /**
+     * Adds the widget to the list of widgets stored in the {@code CardLayout}
+     *
+     * @param element The {@code UIWidget} to add
+     * @param hint A hint as to how the widget should be laid out - may be null (and null values should be handled).
```

Comment:
How should this be documented given that the `hint` parameter is not actually used here? I'm not exactly sure what is the point of this constructor given that the difference between this constructor and the other `addWidget` constructor is only that it bypasses the check for an id, allowing widgets without an id to be added. However, widgets without an id can never be displayed in the CardLayout.